_harp/apply/_london.md

```diff
@@ -1,6 +1,6 @@
 <h2 align='center'>SELECTION PROCESS</h2>
 
-We will review applications and offer interviews at the end of July, including a guranteed interview to all candidates who have completed the course prerequisites. Interviews will be conducted in-person (or via Google hangouts) in early August. We will select 16 people for the course beginning in October.
+From 31st July we will review applications and offer interviews, including a guaranteed interview to all candidates who have completed the course prerequisites. Interviews will be conducted in-person (or via Google hangouts) in early August. We will select 16 people for the course beginning in October.
```

Comment:
Two different ways of writing dates in the same file:
> 31st July

> 18 February

We should pick one style and stick to it.